Class/C-Array-Methods/exercise4.js

```diff
@@ -4,12 +4,18 @@
 let numbersList = [5, 3, 15, 8, 24, 31, 12];
 
 function doubleValue(x){
-    return x * 2;
+    //return x * 2;
 }
 
 let doubledNumbers = [];
 
+for (let i - 0; i < namesList.length; i++){
+    let number - numbersList[i];

```

Comment:
`-` should be `=` in order to assign the value however you don't need a for loop - see below.